README.md

```diff
@@ -61,18 +61,25 @@ Redeploy, open Content Studio, edit the site item, and on the app where you see
 
 ## Overview
 
-The lib adds a proxy controller (_/lib/frontend-proxy/proxy.js_) that requires a [controller mapping](https://developer.enonic.com/docs/xp/stable/cms/mappings) in the host app's _site.xml_ (see "installation" above). Set the proxy up so it matches on
-existing content items of selected type on path `<sitePath>/X/Y/Z` - so, usually <match>type. The proxy sends requests to `<frontendUrl>/X/Y/Z`, and handles any errors or returns the response so that is rendered as a preview in content studio.
+The lib adds a proxy controller (_/lib/frontend-proxy/proxy.js_) that requires
```

Comment:
For consistency, I think the files should be moved to`/lib/nextjs-proxy/proxy.js` too. Is that suitable in this issue, or a different one?